src/MCPClient/lib/clientScripts/removeUnneededFiles.py

```diff
@@ -2,21 +2,32 @@
 
 import os
 import sys
+import shutil
 
 # databaseFunctions requires Django to be set up
 import django
 django.setup()
 # archivematicaCommon
 from custom_handlers import get_script_logger
 from databaseFunctions import fileWasRemoved
+import ConfigParser
 
-REMOVEABLE_FILES = ["Thumbs.db", "Icon", u"Icon\u000D"]
+clientConfigFilePath = '/etc/archivematica/MCPClient/clientConfig.conf'
+config = ConfigParser.SafeConfigParser()
+config.read(clientConfigFilePath)
+try:
+    removableFiles = [e.strip() for e in config.get('MCPClient', 'removableFiles').split(',')]
+except:
```

Comment:
Sure -- I actually copied the syntax from https://github.com/artefactual/archivematica/blob/ae0c0f1c9a55cd19f32105179d7908cb02939cfd/src/dashboard/src/components/helpers.py and https://github.com/artefactual/archivematica/blob/ae0c0f1c9a55cd19f32105179d7908cb02939cfd/src/archivematicaCommon/lib/elasticSearchFunctions.py since that seemed to be how you're handling this config file elsewhere :)